CHANGELOG

```diff
@@ -2,6 +2,8 @@
 --- PyFMI-2.9.7 ---
     * Added an argument to ResultDymolaBinary to allow for reading updated
       data from the loaded file.
+    * Added option to allow for synchronizing simulation with (scaled)
+      real-time.
```

Comment:
Would it be more clear if we mention the name of the new option? Like "Added a new option 'synchronize_simulation' to ..."?

Reply:
Thanks, fixed!